src/shared/theme/theme.ts

```diff
@@ -72,17 +78,26 @@ export const theme = extendTheme({
       marginTop: 1 * 8,
       marginBottom: 2 * 8,
     },
-    "body-lg": {
+    "title-lg": {
       fontSize: "18px",
       fontWeight: "600",
       lineHeight: "166%",
       color: "inherit",
       marginTop: 1 * 8,
       marginBottom: 1 * 8,
     },
-    "body-md": {
+    "body-lg": {
+      fontFamily: "Inter",
       fontSize: "18px",
+      fontStyle: "normal",
+      fontWeight: "400",
       lineHeight: "155%",
+      marginTop: 1 * 8,
+      marginBottom: 1 * 8,
+    },
+    "body-md": {
+      fontSize: "16px",
+      lineHeight: "150%",
```

Comment:
the old "Body-lg" is used in figma as "title-lg". This aligns the webapp with figma a bit better. 

Same reasoning for the "bossBlue" rename